contracts/osmosis-liquidity-helper/src/contract.rs

```diff
@@ -88,6 +88,14 @@ pub fn execute_balancing_provide_liquidity(
     // Assert that only native coins are sent
     assert_only_native_coins(&assets)?;
```

Comment:
Could remove this now since the check below would fail on the same conditions.

Reply:
Ill go for [this](https://github.com/apollodao/liquidity-helpers/pull/13#issuecomment-1351331886) solution instead

Reply:
fixed